docs/apidiffs/current_vs_latest/opentelemetry-exporter-logging-otlp.txt

```diff
@@ -1,2 +1,5 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+***! MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.exporter.logging.otlp.OtlpJsonLoggingMetricExporter  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	---! REMOVED METHOD: PUBLIC(-) io.opentelemetry.sdk.common.CompletableResultCode flush()
```

Comment:
Hrm. Since this is in a stable module, should we keep this method and just deprecated it?